.github/actions/inject-issue-id-action/README.md

```diff
@@ -0,0 +1,60 @@
+# 🎟️ Add JIRA ticket/issue to commit message body
+
+Action to add a JIRA ticket/Issue-ID line to the body of a commit message.
+
+## inject-issue-id-action
+
+## Usage Example
+
+Pass a JSON key/value lookup table to the action, either directly as a string,
+or by passing in the contents of a variable (set either at the repository or
+organisation level).
+
+```yaml
+- name: "Check/Add commit message JIRA ticket"
+  uses: ModeSevenIndustrialSolutions/issue-id/.github/actions/inject-issue-id-action@main
+  with:
+      issue_id_lookup_json: ${{ vars.ISSUE_ID_LOOKUP_JSON }}
+```
+
+In the example above, the repository passes in the JSON data contained in
+a variable configured called ISSUE_ID_LOOKUP_JSON.
+
+Set/define this for the repository under:
+
+GitHub Repository -> Settings -> Secrets and variables -> Actions -> Variables
+
+## Inputs
+
+The action does not have any default values and both input parameters are mandatory.
+
+<!-- markdownlint-disable MD013 -->
+
+| Variable Name        | Required | Default | Description                         |
+| -------------------- | -------- | ------- | ----------------------------------- |
+| ISSUE_ID_LOOKUP_JSON | True     | N/A     | JSON array of key/value pairs       |
+| INJECT               | False    | True    | When set false, checks for presence |
+
+<!-- markdownlint-enable MD013 -->
+
+## Outputs
+
+| Variable Name | Description                                   |
+| ------------- | --------------------------------------------- |
+| PRESENT       | Set true when ticket string found or injected |
+
+## Limitations
+
+This action uses ${{ github.actor }} as the key to lookup, but also reports
```

Comment:
This ${{ github.actor }} doesn't render in markdown; needs to be somehow quoted verbatim.